### PR TITLE
Fix inline resolver for open api generator

### PR DESCRIPTION
**Pull request by altro3**
This is more corect solution for openapi generator problem. This is fix for bug in openap genertor: problem with parameters, when it collection of enum - openapi generator lost this enum and don't generate it. Yesterday I fixed it with internal enums inside client interfaces and controller interfaces, this solution works, but it's not entirely correct. It is necessary to create separate files for such enums so that this logic does not differ from the rest. Today I fixed` openapi-generator/src/main/java/io/micronaut/openapi/generator/MicronautInlineModelResolver.java` - Now it handles this case correctly.

### CODE REVIEW

File: openapi-generator/src/test/java/io/micronaut/openapi/generator/AbstractMicronautCodegenTest.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ protected String generateFiles(MicronautCodeGenerator<?> codegen, String configP
         } catch (IOException e) {
             fail("Unable to create temporary directory for output");
         }
-        output.deleteOnExit();
+//        output.deleteOnExit();
```
Comment (sdelamo) [inline openapi-generator/src/test/java/io/micronaut/openapi/generator/AbstractMicronautCodegenTest.java]: is this commented on purpose.

**Review by sdelamo — CHANGES_REQUESTED** [review]
Can you add a PR description about what this change is about?

File: openapi-generator/src/test/java/io/micronaut/openapi/generator/AbstractMicronautCodegenTest.java (hunk 1 above)
Comment (altro3) [reply openapi-generator/src/test/java/io/micronaut/openapi/generator/AbstractMicronautCodegenTest.java]: fixed

**Comment by altro3** [comment]
> Can you add a PR description about what this change is about?

done

**Review by sdelamo — APPROVED** [review]

**Pull request merged**